web/packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snowbridge/api",
-  "version": "0.1.46",
+  "version": "0.1.47",
```

Comment:
This clashes with my PR https://github.com/Snowfork/snowbridge/pull/1450. May need to sync here.